.travis.yml

```diff
@@ -5,6 +5,7 @@ install:
 script: bash -ex .travis-opam.sh
 env:
   matrix:
-    - OCAML_VERSION=4.00
-    - OCAML_VERSION=4.01
-    - OCAML_VERSION=4.02
+    - OCAML_VERSION=4.02 PACKAGE=cdrom
```

Comment:
We could put `PACKAGE=cdrom` in the global env and `allow_fail` the `OCAML_VERSION-4.05`. But it is not strictly necessary for this package.